README.md

```diff
@@ -26,6 +26,12 @@ pip install hiddenlayer-sdk[hf]
 
 ## Getting Started
 
+If you are using the EU region, you will need to set the `HL_AUTH_URL` environment variable to the following:
+
+```bash
+export HL_AUTH_URL=https://api.eu.hiddenlayer.ai
+```
+
```

Comment:
Also, I think we probably need to make sure to describe configuring the host to the EU endpoint as well which would be set to "https://api.eu.hiddenlayer.ai"